Thor Method.py

```diff
@@ -0,0 +1,224 @@
+#
+# My Thoughts and Process
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# First off, taking a *step back* and drawing out the xy plane and the directions the are required was helpful. 
+# 
+#  .. image:: Power_of_thor_pic_2.png
+#  
+# However, without the original code telling me exactly what the directions did, I had no idea of knowing that the initial Thor position never changed. Keeping notes helped me track how past codes I wrote worked and didn’t work. Taking notes was helpful then just trying to type in code without much guidance. It will take me a while to look and gather before trying code. 
+# I started with trying to figure out Thor's position in comparison to light's position but did it have in common with directions.
+#
+# .. image:: Power_of_thor_pic_4.png
+#
+# I started with a while loop but noticed it only worked for a straight line movement. Next i tried if loops. That just got messy. Last i tried a for loop, and incorprated Thor's position getting updated. 
+#
+# .. image:: Power_of_thor_pic_5.png
+# .. image:: Power_of_thor_pic_6.png
+#
+# Errors in My Code
+# ^^^^^^^^^^^^^^^^^
+# Due to all the if statements I had and built in while loops with only work 75% of the time. When I have time I may go back and clean up this code to a similar form and able to work 100% of the time. 
+# 
+# Looking Back
+# ^^^^^^^^^^^^^^^^^
+# Looking back, if I was going to do the game in C++, I really should have done the codechat in C++ also. I am learning Python now to make this a little easier on me so I don't have to switch back and forth. I would copy and paste the C++ code I used in the game; however, python doesn't care for C++ and keeps putting errors at the top. 
+#
+# Soultion: Second Try with Python
+# ----------------------------------------
+# With this second try, I am going to work in `Python <https://wiki.python.org/moin/BeginnersGuide>`_.
+#
+# My Thoughts and Process
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# The whole idea is to get Thor to a power source using the xy plane. So if Thor is at (0,3) and the light is at (1,3) then Thor needs to move to the right one. Since postive x is to the right (**East**) and postive y is down (**South**). Another given is that x is 0-40 and y is 0-18 and the coodinates start at the top left of the boundry. 
+#
+# However, as I found out this my first try, Thor's position isn't being tracked, just his intial position. This needs to taken into account. We already found out with try 1 that loops aren't the best way to complete this code. Maybe we can use math instead. 
+#
+#  Example: 
+#
+#   If Thor is at (0,0) and the light is at (40,0). We know just by looking at this that Thor is -40 from the light in the x direction. So to solve this we need to add 40 to Thor's intial position.
+#
+print(initial_tx, initial_ty, light_x, light_y)  
+#
+#| The above code gets the program to show us where Thor and the light start.
+#
+x_steps = [int]  # this is going to track our difference in Thor's x and Light's x.
```

Comment:
These two lines make no sense to me. Also, `x_steps` and `y_steps` are reassigned in line 111-112, so these two statements are also unnecessary.

Reply:
I was keeping track of my thought process. this one didn't work